workshop/content/advanced/resource-types/key-concepts/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: 'Key Concepts'
+date: 2021-11-16T20:44:31Z
+weight: 310
+---
+
+### Getting started
```

Comment:
Can we move this section onto `workshop/content/advanced/resource-types/_index.md` Its rather short and I feel it doesn't need own page. After introduction it may be nice to have a key concepts on the same page.

Reply:
Sure - thank you!  Will do!

Reply:
Done